### DISCUSSION TITLE
How to trigger draw event remotely?

### DISCUSSION THREAD

**Discussion by moyjohnsen** [szimek/signature_pad#628] [General]
I would like to know if it is possible to trigger the draw event from outside of the canvas. Something like so: 

`somediv.addEventListener('touchstart', function(){
     startdrawing(someotherdiv);
})`

I am doing this because I have a dynamically created div, and the `new signaturePad` call assigned to that div works on desktop browsers, but not on mobile. So I would like to call the draw function manually on a touch event.

**Comment by UziTech** [comment]
You can try triggering the event on the other div

```js
somediv.addEventListener('touchstart', (event) => {
  someotherdiv.dispatchEvent(new TouchEvent('touchstart', event));
});
```
